README.md

```diff
@@ -59,6 +59,36 @@ Follow the steps below in order to setup and view the Jupyter notebooks:
     jupyter lab --config=config.py
     ```
 
+5. Optional [Autoformatting](https://jupyterlab-code-formatter.readthedocs.io/en/latest/index.html) for jupyter notebooks using jupyter lab:
+
+    ```bash
+    pip install jupyterlab_code_formatter
+    pip install black
+    cd examples
+    jupyter lab --config=config.py
+    Settings < Advanced Settings Editor < Jupyterlab Code Formatter
+    ```
+
+    Two panels should open up. The first (left) panel will be 'System Defaults' and the second (right) panel will be 'User Preferences'.
+
+    Paste the following in User Preferences:
+
+    ```
+    {
+        "formatOnSave": true,
+
+        "preferences": {
+            "default_formatter": {
+                "python": "black",
+                "R": "styler"
+            }
+        }
+    }
```

Comment:
There must be a way to set the User Preferences inside config.py to avoid having to copy/paste the content.